src/core/tests/file_util.cpp

```diff
@@ -469,3 +457,176 @@ TEST(file_util, unicode_path_cast_from_wstring_to_char8_t) {
               ov::util::Path(L"~/狗/ǡ୫ԩϗ/にほ/ąę/ど/௸ඊƷ/狗33.txt").generic_u8string());
 #endif
 }
+
+class CutAndroidPathTests : public ::testing::TestWithParam<std::tuple<ov::util::Path, ov::util::Path>> {};
+
+TEST_P(CutAndroidPathTests, HandlesStringPaths) {
+    ov::util::Path file_name = std::get<0>(GetParam());
+    ov::util::Path expected = std::get<1>(GetParam());
```

Comment:
```suggestion
    const auto& [file_name, expected] = GetParam();
```
In cpp17 it should work.

Reply:
Done